docs/resources/aom_alarm_rule_v4.md

```diff
@@ -0,0 +1,424 @@
+---
+subcategory: "Application Operations Management (AOM)"
+layout: "huaweicloud"
+page_title: "HuaweiCloud: huaweicloud_aom_alarm_rule_v4"
+description: |-
+  Manages an AOM cloud alarm rule resource within HuaweiCloud.
+---
+
+# huaweicloud_aom_alarm_rule_v4
+
+Manages an AOM cloud alarm rule resource within HuaweiCloud.
+
+## Example Usage
+
+### Create an event alarm rule with system event
+
+```hcl
+variable "alarm_rule_name" {}
+variable "action_rule_id" {}
+
+resource "huaweicloud_aom_alarm_rule_v4" "test" {
+  name        = var.alarm_rule_name
+  type        = "event"
+  description = "test"
+  enable      = true
+
+  alarm_notifications {
+    notification_type         = "direct"
+    notification_enable       = true
+    bind_notification_rule_id = var.action_rule_id
+  }
+
+  event_alarm_spec {
+    event_source = "CCE"
+    alarm_source = "systemEvent"
+
+    monitor_objects = [
+      {
+        event_name = "扩容节点超时##ScaleUpTimedOut;数据卷扩容失败##VolumeResizeFailed"
```

Comment:
does the enent_name must contain Chinese? How to use it if the customer does know Chinese?

Reply:
It's required.